watson/watson.py

```diff
@@ -251,7 +251,12 @@ def add(self, project, from_date, to_date, tags):
         frame = self.frames.add(project, from_date, to_date, tags=tags)
         return frame
 
-    def start(self, project, tags=None, restart=False, gap=True):
+    def start(self, project, tags=None, restart=False, start_at=None, gap=True):
+        if start_at is not None and not gap:
```

Comment:
I noticed now that I could use the `MutuallyExclusiveOption` in the click decorator instead of this if statement. The only drawback would be that I didn't see a way to specify a custom error message which might be helpful, but it would still probably be better to handle this the same as other mutually exclusive options in watson and use the decorator. Let me know what you prefer.

Reply:
IMO we should use the decorator when applicable.

Reply:
Thanks, I changed this to use the decorator in my latest commit. Note the small difference that the previous only raised an error if `--at` was used together with `--no-gap`, while  the click decorator raises and error when used together with either `--gap` or `--no-gap` (it still works with the default parameter value of `gap=True`, it is just not possible to use the flag directly).

Reply:
Also just a note that there is apparently a click addon for options groups now https://github.com/click-contrib/click-option-group, including mutual exclusiveness and grouping in the help message, might be something worth checking out in the future.